bin/i18n/resources/pegasus.rb

```diff
@@ -11,12 +11,14 @@ def self.sync_in
         HourOfCode.sync_in
         Markdown.sync_in
         Mobile.sync_in
+        Emails.sync_in
```

Comment:
Unless there's a dependency on the order components are synced, we are trying to keep the calls ordered alphabetically.